src/components/organisms/NoteList/NoteList.tsx

```diff
@@ -128,6 +140,8 @@ const NoteList = ({
   setSort,
   trashOrPurgeCurrentNote,
 }: NoteListProps) => {
+
+  const [liststyle, setListStyle] = useState(false);
```

Comment:
The state should be kept by preferences store. Please add `noteListStyle` prop to preferences hook store.

Reply:
Hey,
Apologies for late response.
I am having difficulty understanding how to add to preferences hook store.
Can you please help.

Reply:
It is quite similar to normal state hook. The difference is that the store is singleton and is passed via React's context. So you can access the hook state from any component.

Here's instruction
1. Add a new property to `Preferences` interface in `src/lib/preferences.ts`. I would name the key `general.noteListStyle: 'default' | 'compact'`
2. Also update `basePreferences` to set default value, `default`. 

Then, you can implement it in the note list like this
```ts
import { usePreferences } from '../../lib/preferences'
...

const NoteList = () => {

  ...
  const { preferences, setPreferences } = usePreferences()

  // How to access the option
  const listStyle = preferences['general.noteListStyle']

  // How to update the option
  const updateListStyle = useCallback((listStyle: 'default' | 'compact') => {
    setPreferences({ 'general.noteListStyle': listStyle })
  }, [setPreferences])
  ...
}

```


And please fix the conflicts. I've changed quite lots of code in v0.4.0